mlflow/models/evaluation/base.py

```diff
@@ -1650,8 +1677,7 @@ def fn(model_input):
             ``data`` is a :py:class:`mlflow.data.dataset.Dataset` that defines targets,
             then ``targets`` is optional.
 
-        predictions: Optional. The name of the column that contains model outputs. There are two
```

Comment:
Update this line to no longer indicate that `predictions` is strictly required. Still provide guidance about how to use it, if desired

Reply:
Can you also update 

```
When ``model`` is not specified and ``data`` is a pandas dataframe. The
              ``predictions`` should be the name of the column in ``data`` that
              contains model outputs.
```

This also implies that predictions is required.

Reply:
Absolutely! Updated!